tests/conftest.py

```diff
@@ -199,3 +199,39 @@ def df_normal_dist():
     df.columns = ["var"]
 
     return df
+
+
+@pytest.fixture(scope="module")
+def df_pred():
```

Comment:
lets more this df into a conftest inside the test_prediction module.

Reply:
Done. What's the benefit of having a separate conftest file?